_pages/install/schematics.md

```diff
@@ -138,6 +138,9 @@ During the initial set up of your storefront using schematics, you have the opti
 - `@spartacus/storefinder` includes the [{% assign linkedpage = site.pages | where: "name", "store-locator.md" %}{{ linkedpage[0].title }}]({{ site.baseurl }}{% link _pages/dev/features/store-locator.md %}) feature.
 - `@spartacus/tracking` includes the [{% assign linkedpage = site.pages | where: "name", "tag-management-system.md" %}{{ linkedpage[0].title }}]({{ site.baseurl }}{% link _pages/dev/features/tag-management-system.md %}) feature and the [{% assign linkedpage = site.pages | where: "name", "personalization-setup-instructions-for-spartacus.md" %}{{ linkedpage[0].title }}]({{ site.baseurl }}{% link _pages/install/personalization-setup-instructions-for-spartacus.md %}).
 - `@spartacus/user` includes the Account and Profile features. The Account feature contains components such as the login form, and also exposes the general method for getting user details. The Profile feature is responsible for functionality such as closing an account, updating a profile, updating an email, updating a password, resetting a password, and registering. It is highly recommended to install both of these features.
+- `@spartacus/checkout/base` - includes basic checkout functionalities, which is more catered to business to consumer (b2c).
+- `@spartacus/checkout/b2b` - includes the basic checkout functionalities from base, and additional checkout flow that is catered to business to business (b2b).
+- `@spartacus/checkout/scheduled-replenishment` - includes both the base and b2b checkout functionalities, but it allows the user to either place an order or schedule a replenishment order.
```

Comment:
We should list only `@spartacus/checkout`, as customer's can't run the `@spartacus/checkout/base` (or any other variant).